scrapers/bugzilla.py

```diff
@@ -49,7 +49,7 @@ def get_bugzilla_bugs(self, start_date):
             LEFT JOIN bugzilla.profiles AS assigned ON bugs.assigned_to = assigned.userid
             LEFT JOIN bugzilla.profiles AS reported ON bugs.reporter = reported.userid
             LEFT JOIN bugzilla.profiles AS qa ON bugs.qa_contact = qa.userid
-            WHERE classifications.name = 'Red Hat' AND bugs.creation_ts > '{}'
+            WHERE classifications.name = 'Red Hat' AND bugs.lastdiffed > '{}'
```

Comment:
I think we actually want to use `delta_ts` here.

Reply:
They are the same for all bugs :)

Reply:
Ah I see. This is the value that's exposed in the API, so I think we should use that one instead.

Reply:
Agreed. Changed it :)